lib/ActivityStreams.js

```diff
@@ -0,0 +1,144 @@
+/* globals NewTabURL, EventEmitter, XPCOMUtils */
+
+"use strict";
+
+const {Cu} = require("chrome");
+const {data} = require("sdk/self");
+const {PageMod} = require("sdk/page-mod");
+const {PlacesProvider} = require("lib/PlacesProvider");
+
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+Cu.import("resource:///modules/NewTabURL.jsm");
+
+XPCOMUtils.defineLazyGetter(this, "EventEmitter", function () {
+  const {EventEmitter} = Cu.import("resource://devtools/shared/event-emitter.js", {});
+  return EventEmitter;
+});
+
+let ActivityStreams = function (options) {
+  console.debug(`ActivityStreams.init`);
```

Comment:
Nit, as the constructor is not referenced as variable anywhere, you can just write: 
JS```
function ActivityStreams(options = {pageURL: data.url("content/activity-streams.html"}) {
```